doc/source/whatsnew/v0.20.0.txt

```diff
@@ -587,3 +587,5 @@ Bug Fixes
 - Bug in ``Series.replace`` and ``DataFrame.replace`` which failed on empty replacement dicts (:issue:`15289`)
 - Bug in ``pd.melt()`` where passing a tuple value for ``value_vars`` caused a ``TypeError`` (:issue:`15348`)
 - Bug in ``.eval()`` which caused multiline evals to fail with local variables not on the first line (:issue:`15342`)
+
```

Comment:
don't put whatsnew at the very end as they cause conflicts. the point of the empty space elsewhere is for you to use